src/main/java/me/moodcat/eemcsdata/parser/ParseData.java

```diff
@@ -0,0 +1,101 @@
+package me.moodcat.eemcsdata.parser;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import me.moodcat.eemcsdata.JsonData;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * A class to parse the features from the EEMCS data and ouput a .JSON file with only the features
+ * we are using.
+ */
+public class ParseData {
+
+    /**
+     * The objectMapper that maps the old file into the new file.
+     */
+    private ObjectMapper objectMapper;
+
+    /**
+     * The data class where we store the data.
+     */
+    private JsonData result;
+
+    /**
+     * This class makes a new objectMapper wich mapps the JSOn objects.
+     */
+    public ParseData() {
+        this.objectMapper = new ObjectMapper();
+    }
+
+    /**
+     * Get the latest result class.
+     *
+     * @return The JsonData class result.
+     */
+    public JsonData getResult() {
+        return this.result;
+    }
+
+    /**
+     * A method to parse the data and ouput the data to work with it.
+     *
+     * @param input
+     *            - the path to the input file
+     * @param output
+     *            - the path where to output the file.
+     * @return A json data class to store the data. Returns null if an I/O exception occured.
+     */
+    public JsonData parseFile(final String input, final String output) {
+        try (InputStream in = new FileInputStream(input)) {
+            this.result = this.objectMapper.readValue(in, JsonData.class);
+
+            this.objectMapper.writeValue(new File(output), this.result);
+            return this.result;
+        } catch (final IOException e) {
+            e.printStackTrace();
+        }
+        return null;
```

Comment:
Returning null will cause an `NullPointerException`. This is not any better than an `IOException`. The `IOException` gives more information of why the parser failed. So just rethrow the `IOException`, and otherwise return the result.